### PR TITLE
README.ES.md -> 00_before_we_start

### PR DESCRIPTION
**Pull request by zanezhub**
### New files 48cbc92 
I translated the [README.md of 00_before_we_start](https://github.com/rust-embedded/rust-raspberrypi-OS-tutorials/blob/master/00_before_we_start/README.md). 

### Corrections eb2a88f
I made a few corrections to the [main README.ES.md](https://github.com/rust-embedded/rust-raspberrypi-OS-tutorials/blob/master/README.ES.md) and to the 00_before_we_start README.ES.md.

### CODE REVIEW

File: 00_before_we_start/README.ES.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+```
+
+# Resumen
+
+Para un subsistema lógico del `núcleo (kernel)`, el código correspondiente puede ser distribuido sobre diferentes localizaciones físicas. Aquí un ejemplo para el subsistema de memoria:
+
+- `src/memory.rs` y `src/memory/**/*`
+  
+  - Código común que es independiente de la arquitectura del procesador de destino y las características de `BSP`.
+    - Ejemplo: Una función a un pedazo cero de memoria.
```
Comment (eldruin) [inline 00_before_we_start/README.ES.md:82]:
```suggestion
    - Ejemplo: Una función para poner a cero un trozo de memoria.
```

File: 00_before_we_start/README.ES.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  - Las interfaces para el subsistema de la memoria que son implementados por código `arch` o `BSP`.
```
Comment (eldruin) [inline 00_before_we_start/README.ES.md:83]:
```suggestion
  - Las interfaces para el subsistema de la memoria que son implementados por código de `arch` o `BSP`.
```

File: 00_before_we_start/README.ES.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    - Ejemplo: Una interface `MMU` que define prototipos de función `MMU`.
```
Comment (eldruin) [inline 00_before_we_start/README.ES.md:84]:
```suggestion
    - Ejemplo: Una interfaz `MMU` que define prototipos de función de `MMU`.
```

File: 00_before_we_start/README.ES.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  - Ejemplo: Implementación de la interfaz `MMU` para la arquitectura `__arch_name__`.
+
+Desde una perspectiva de *namespace*, el código del subsistema de **memoria** vive en:
+
+- `crate::memory::*`
+- `crate::bsp::memory::*`
+
+# Flujo de Boot / Boot flow
+
+1. La punto de entrada del núcleo (kernel) es la función `cpu::boot::arch_boot::_start()`.
```
Comment (eldruin) [inline 00_before_we_start/README.ES.md:103]:
```suggestion
1. El punto de entrada del núcleo (kernel) es la función `cpu::boot::arch_boot::_start()`.
```

File: 00_before_we_start/README.ES.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+   - Está implementada en `src/_arch/__arch_name__/cpu/boot.s`.
```
Comment (eldruin) [inline 00_before_we_start/README.ES.md:104]:
```suggestion
   - Está implementado en `src/_arch/__arch_name__/cpu/boot.s`.
```

**Review by eldruin — COMMENTED** [review]
Looks pretty good to me!
Just a few improvements/corrections.

**Comment by zanezhub** [comment]
I made a new commit  afe4c2d with the requested changes/corrections

**Review by eldruin — APPROVED** [review]
Looks good to me, thank you!

**Pull request merged**